src/test/resources/recordingsHelper.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+interface TestUsingRecording {
+    (expectation: string, recordingNames: string[], assertion?: () => void): Mocha.ITest;
+    (expectation: string, recordingNames: string[], assertion?: (done: MochaDone) => void): Mocha.ITest;
+    only(expectation: string, recordingNames: string[], assertion?: () => void): Mocha.ITest;
+    only(expectation: string, recordingNames: string[], assertion?: (done: MochaDone) => void): Mocha.ITest;
+    skip(expectation: string, recordingNames: string[], assertion?: () => void): void;
+    skip(expectation: string, recordingNames: string[], assertion?: (done: MochaDone) => void): void;
+}
+
+export interface IRecordingConsumer {
+    loadRecordingFromName(recordingName: string): Q.Promise<void>;
+}
+
+/* This class makes it easy to create a test using a recording. Recommended usage is:
+     const testWithRecordings = new RecordingsHelper(() => recordingConsumer).test;
+     testWithRecordings("expects to do some test thing",
+    [
+        "path/to/recording",
+        "path/to/recording"
+    ], () => {
+        // test code here
+    });
+*/
+export class RecordingsHelper {
+    public test: TestUsingRecording;
+
+    constructor(private getRecordingConsumer: () => IRecordingConsumer) {
+        this.initializeTest();
+    }
+
+    private initializeTest(): void {
```

Comment:
I choose to do it this way because it needs to support the TestUsingRecording interface which is a function + 2 methods.
We could also do it with a member function, but then we'd need to cast it to <any> to assign the sub-functions, and also make sure we don't use "this" inside that member, given that we are going to pass it around, and bind will probably break the sub-member function.

@dlebu I'm fine with either approach, if you think that the member function is better I'll change it to that.